debug/examples/shop.json5

```diff
@@ -49,13 +49,15 @@
     listPadding: {
         x: 18.86,
         y: 15.09,
+        larger: 1.8,
```

Comment:
Is this outer or inner padding, what is larger than what?

Reply:
Yeah, that needs to be clearer. TY

Reply:
a9c902c